### PR TITLE
Update list of images and descriptions

### PR DESCRIPTION
**Pull request by magdapoppins**
- Update list of supported images
- Create a new list with more detailed descriptions

Thoughts: should we clear out some of the tensorflow suggestions? Also, I added a bool 'recommended', maybe only have that checked for a newer stable version (I set it to 2.2.0)?

HOX: I yeeted the `descriptions` part so that we can instead use richer descriptions, this change (1 line) needs to be in roi whenever this thing is mergéd.

### CODE REVIEW

File: images.yaml [hunk 1]
```diff
@@ -92,8 +92,25 @@ suggestions:
...
+  - tensorflow/tensorflow:2.2.0-py3
+  - tensorflow/tensorflow:2.2.0-gpu
+  - tensorflow/tensorflow:2.2.0-gpu-py3
+  - tensorflow/tensorflow:2.3.0rc2-gpu
+  - tensorflow/tensorflow:2.3.0rc1-gpu
+  - valohai/keras:2.0.0-tensorflow1.0.1-python3.6-cuda8.0-cudnn5-devel-ubuntu16.04
+  - valohai/keras:2.0.0-theano0.9.0rc4-python3.6-cuda8.0-cudnn5-devel-ubuntu16.04
+  - valohai/darknet:b61bcf5-cuda8.0-cudnn5-devel-ubuntu16.04
 
 descriptions:
```
Comment (magdapoppins) [inline images.yaml:93]: Leaving these here until we move to using details in roi.

File: images.yaml [hunk 2]
```diff
@@ -92,8 +92,25 @@ suggestions:
   - ufoym/deepo:all-py36-cpu
   - ufoym/deepo:all-py27
   - ufoym/deepo:all-py27-cpu
```
Comment (akx) [inline images.yaml]: I think we could get rid of all the py27-based caches and suggestions?

File: images.yaml [hunk 3]
```diff
@@ -92,8 +92,25 @@ suggestions:
...
+  - pytorch/pytorch:1.5.1-cuda10.1-cudnn7-runtime
+  - pytorch/pytorch:1.4-cuda10.1-cudnn7-devel
+  - tensorflow/tensorflow:1.8.0-devel
   - tensorflow/tensorflow:1.13.1-gpu-py3
   - tensorflow/tensorflow:1.13.1-py3
+  - tensorflow/tensorflow:1.15.0-gpu-py3
+  - tensorflow/tensorflow:1.15.2-gpu-py3
+  - tensorflow/tensorflow:2.0.0-gpu-py3
+  - tensorflow/tensorflow:2.0.1-gpu-py3
+  - tensorflow/tensorflow:2.2.0
```
Comment (akx) [inline images.yaml]: Maybe we could actually drop the py2 variants of the newly added libraries too, actually? It's a slightly political decision but I don't really see the reason to hang on to py2 :)

**Review by akx — CHANGES_REQUESTED** [review]

File: images.yaml [hunk 4]
```diff
@@ -90,24 +75,202 @@ aliases:
...
+      - 'torch 7'
+      - 'keras'
+      - 'mxnet'
+      - 'cntk'
+      - 'chainer'
+      - 'theano'
+      - 'lasagne'
+      - 'caffe 1.0.0'
+      - 'caffe2'
+  ufoym/deepo:all-py36-cpu:
```
Comment (magdapoppins) [inline images.yaml:196]: I hope I got the tags right on this, but there were no versions stated and not sure if cuDNN can be on a non-gpu variant?

Comment (magdapoppins) [reply images.yaml:196]: Not sure how this should be read... => https://github.com/ufoym/deepo#Comparison

Comment (akx) [reply images.yaml:196]: cuDNN is the CUDA deep-learning library, so anything with it is GPU-flavored.

**Review by akx — APPROVED** [review]

**Pull request merged**